### PR TITLE
ZF3 compatibility

### PR DESCRIPTION
**Pull request by thomasvargiu**
ZF3 compatibility

### CODE REVIEW

**Comment by shipleyr** [comment]
@juriansluiman Any chance of getting this merged?

File: src/ZfcAdmin/Module.php [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    Feature\AutoloaderProviderInterface,
+    Feature\ConfigProviderInterface,
+    Feature\BootstrapListenerInterface
+{
+    /**
+     * @{inheritdoc}
+     */
+    public function getAutoloaderConfig()
+    {
+        return array(
```
Comment (samsonasik) [inline src/ZfcAdmin/Module.php]: short array syntax also for consistency

Comment (thomasvargiu) [inline src/ZfcAdmin/Module.php]: @samsonasik Done!

**Comment by mihailgoosbg** [comment]
Any idea when can we expect pull request getting merged?

**Comment by juriansluiman** [comment]
I am not working with ZF2 (ZF3) anymore, so I can't test it. Can somebody confirm? I'll merge accordingly.

Besides, if anyone is available for maintaining ZfcAdmin, please request this to any Zf-commons owner, they will probably grant you access then.

**Comment by thomasvargiu** [comment]
I'm using this branch on ZF2, not yet in ZF3. Anyone?

**Comment by thomasvargiu** [comment]
@juriansluiman I finally using it with ZF3!

**Comment by thomasvargiu** [comment]
@juriansluiman Can we merge this PR?

**Comment by thomasvargiu** [comment]
ping @samsonasik

**Comment by samsonasik** [comment]
I don't have merge access for it, sorry.

**Comment by thomasvargiu** [comment]
So, ping @juriansluiman :)

File: Module.php [hunk 2]
```diff
@@ -39,96 +39,4 @@
...
-            || $controller->getEvent()->getResult()->terminate()
-        ) {
-            return;
-        }
-
-        $layout     = $config['zfcadmin']['admin_layout_template'];
-        $controller->layout($layout);
-    }
-}
+include_once __DIR__ . '/src/ZfcAdmin/Module.php';
```
Comment (akrabat) [inline Module.php]: There's no `ZfcAdmin` subdirectory of `src`…

**Comment by Nirzol** [comment]
News about this compatibility ?

**Comment by akrabat** [comment]
Sorry - I completely missed that you'd updated this.

**Comment by Nirzol** [comment]
Thanks I will try this version this week . Thanks for merging !!

**Pull request merged**